<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventory</class>
 <widget class="QWidget" name="Inventory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>풀필먼트 시스템 - 재고 관리</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>풀필먼트 시스템 > 재고 관리</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="adminLabel">
       <property name="text">
        <string>관리자: 홍길동</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="mainFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="menuList">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentRow">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>대시보드</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>장치 관리</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>창고 환경</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>재고 관리</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>유통기한 관리</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>입출입 관리</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="contentLayout">
        <item>
         <widget class="QGroupBox" name="storageChartGroup">
          <property name="title">
           <string>창고별 재고 현황 (그래프)</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="chartPlaceholder">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>150</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: #f0f0f0; border: 1px dashed #aaa;</string>
             </property>
             <property name="text">
              <string>[창고별 적재량 막대 그래프]</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="storageA">
             <property name="text">
              <string>- 창고A: 29개 (23.2%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="storageB">
             <property name="text">
              <string>- 창고B: 35개 (28.4%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="storageC">
             <property name="text">
              <string>- 창고C: 19개 (15.0%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="storageD">
             <property name="text">
              <string>- 창고D: 41개 (33.4%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="totalLabel">
             <property name="font">
              <font>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>총 124개</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="productListButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: #4272d7; color: white;</string>
               </property>
               <property name="text">
                <string>물품 리스트 보기</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>